domain/src/main/java/org/oppia/domain/profile/ProfileManagementController.kt

```diff
@@ -609,6 +768,14 @@ class ProfileManagementController @Inject constructor(
     }
     val matrix = Matrix()
     matrix.postRotate(rotate.toFloat())
-    return Bitmap.createBitmap(croppedBitmap, /* x= */ 0, /* y= */ 0, cropSize, cropSize, matrix, /* filter= */ true)
+    return Bitmap.createBitmap(
+      croppedBitmap, /* x= */
+      0, /* y= */
+      0,
+      cropSize,
+      cropSize,
+      matrix, /* filter= */
+      true
```

Comment:
Fix these inline comments here and everywhere else in this file.
```
      matrix,
      /* filter= */ true
```

Reply:
Done